spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java

```diff
@@ -38,6 +39,9 @@
 	@Autowired
 	protected KubernetesModuleDeployerProperties properties;
 
+	@Autowired
+	private AdminProperties adminProperties;
+
```

Comment:
I'm not seeing this (either here or CF). What test is this? I can build this project fine, and run the admin fine